src/truststore/_api.py

```diff
@@ -169,6 +169,9 @@ def session_stats(self) -> dict[str, int]:
     def cert_store_stats(self) -> dict[str, int]:
         raise NotImplementedError()
 
+    def set_default_verify_paths(self) -> None:
+        self._ctx.set_default_verify_paths()
```

Comment:
Does this one warrant a changelog entry?

Reply:
Probably, I can make sure it gets added to the 0.10.0 changelog.